src/file.ts

```diff
@@ -1359,10 +1359,11 @@ class File extends ServiceObject<File> {
           md5: this.metadata.md5Hash,
         };
 
+        const doValidation = crc32c || md5;
```

Comment:
Interesting!  Can you explain this change?

Reply:
It's briefly explained in the PR description, but to expand on it:

- If you have validation: crc32 or validation: md5, it will do the according validation
- If you have validation: false, it will return an error message saying "The downloaded data did not match the data from the server."

Result is there's no way to turn validation off, even though the options seem to suggest you can. So, this change makes false actually mean false by separating out "should i do validation" and "result of validation" into two separate vars.

Reply:
It seems like the code from master should work, so maybe it's something else happening? Here are the lines I'm referencing, in order inside of `createReadStream`:

```
let crc32c = true;
let md5 = false;

// ... later ...
} else if (options.validation === false) {
  crc32c = false;
}

// ... later ...
let failed = crc32c || md5;
// this should evaluate to `false`

if (crc32c /* (false) */ && hashes.crc32c) {
  // not run
}

if (md5 /* (false) */ && hashes.md5) {
  // not run
}

if (md5 /* (false) */ && !hashes.md5) {
  // not run
} else if (failed /* false */) {
  // not run
} else {
  // this runs:
  throughStream.end();
}
```

Reply:
Womp womp, you guys are totally right. I had remembered that lost block as having an `if !failed` section. I suspect when I was running into it I must have not had verification: false.

Reply:
Reverted